WWDG not supported on STM32H7 family

**Describe the bug**
I tried to use the `wwdg1` peripheral of the `nucleo_h743zi` board as an hardware backend for the `task_wdt` subsys.

During the compilation I have the following error:
```
/home/nicolas/work/siema/be/VOG/src/zephyrproject/zephyr/drivers/watchdog/wdt_wwdg_stm32.c: In function 'wwdg_stm32_setup':
/home/nicolas/work/siema/be/VOG/src/zephyrproject/zephyr/drivers/watchdog/wdt_wwdg_stm32.c:172:36: error: 'LL_DBGMCU_APB1_GRP1_WWDG_STOP' undeclared (first use in this function); did you mean 'LL_DBGMCU_APB3_GRP1_WWDG1_STOP'?
  172 |   LL_DBGMCU_APB1_GRP1_FreezePeriph(LL_DBGMCU_APB1_GRP1_WWDG_STOP);
      |                                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                    LL_DBGMCU_APB3_GRP1_WWDG1_STOP
/home/nicolas/work/siema/be/VOG/src/zephyrproject/zephyr/drivers/watchdog/wdt_wwdg_stm32.c:172:36: note: each undeclared identifier is reported only once for each function it appears in
[29/42] Building C object zephyr/kernel/CMakeFiles/kernel.dir/sched.c.obj
ninja: build stopped: subcommand failed.
```

**To Reproduce**
1. I modified the board's dts to enable the `wwdg1` peripheral instead of `iwdg1`:
```
diff --git a/boards/arm/nucleo_h743zi/nucleo_h743zi.dts b/boards/arm/nucleo_h743zi/nucleo_h743zi.dts
index 183ba1a85e..e43f392fb4 100644
--- a/boards/arm/nucleo_h743zi/nucleo_h743zi.dts
+++ b/boards/arm/nucleo_h743zi/nucleo_h743zi.dts
@@ -196,6 +196,6 @@ zephyr_udc0: &usbotg_fs {
        };
 };
 
-&iwdg1 {
+&wwdg1 {
        status = "okay";
 };
```

2. Compilation command:
```
west build -p auto -b nucleo_h743zi -d build/nucleo_h743zi/task_wdt_wwdg zephyr/samples/subsys/task_wdt/
```

**Expected behavior**
No compilation error

**Impact**
Cannot use `wwdg` peripheral as hw backend for `task_wdt` subsys.

**Logs and console output**
N/A

**Environment (please complete the following information):**
 - OS: ubuntu 20.04.3 LTS
 - Toolchain: zephyr-sdk-0.13.1
 - 67a265b0df7c13625b52849595c96ab5b2133487

**Additional context**
Using `iwdg` peripheral works.

--- a/drivers/watchdog/wdt_wwdg_stm32.c
+++ b/drivers/watchdog/wdt_wwdg_stm32.c
@@ -168,6 +168,8 @@ static int wwdg_stm32_setup(const struct device *dev, uint8_t options)
 		LL_APB1_GRP2_EnableClock(LL_APB1_GRP2_PERIPH_DBGMCU);
 #elif defined(CONFIG_SOC_SERIES_STM32L0X)
 		LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_DBGMCU);
+#elif defined(CONFIG_SOC_SERIES_STM32G0X)
+		LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_DBGMCU);
 #endif
 #if defined(CONFIG_SOC_SERIES_STM32H7X)
 		LL_DBGMCU_APB3_GRP1_FreezePeriph(LL_DBGMCU_APB3_GRP1_WWDG1_STOP);

--- a/drivers/watchdog/wdt_wwdg_stm32.c
+++ b/drivers/watchdog/wdt_wwdg_stm32.c
@@ -169,7 +169,11 @@ static int wwdg_stm32_setup(const struct device *dev, uint8_t options)
 #elif defined(CONFIG_SOC_SERIES_STM32L0X)
 		LL_APB2_GRP1_EnableClock(LL_APB2_GRP1_PERIPH_DBGMCU);
 #endif
+#if defined(CONFIG_SOC_SERIES_STM32H7X)
+		LL_DBGMCU_APB3_GRP1_FreezePeriph(LL_DBGMCU_APB3_GRP1_WWDG1_STOP);
+#else
 		LL_DBGMCU_APB1_GRP1_FreezePeriph(LL_DBGMCU_APB1_GRP1_WWDG_STOP);
+#endif /* CONFIG_SOC_SERIES_STM32H7X */
 	}
 
 	if (options & WDT_OPT_PAUSE_IN_SLEEP) {

--- a/drivers/watchdog/wdt_wwdg_stm32.c
+++ b/drivers/watchdog/wdt_wwdg_stm32.c
@@ -171,6 +171,8 @@ static int wwdg_stm32_setup(const struct device *dev, uint8_t options)
 #endif
 #if defined(CONFIG_SOC_SERIES_STM32H7X)
 		LL_DBGMCU_APB3_GRP1_FreezePeriph(LL_DBGMCU_APB3_GRP1_WWDG1_STOP);
+#elif defined(CONFIG_SOC_SERIES_STM32MP1X)
+		LL_DBGMCU_APB1_GRP1_FreezePeriph(LL_DBGMCU_APB1_GRP1_WWDG1_STOP);
 #else
 		LL_DBGMCU_APB1_GRP1_FreezePeriph(LL_DBGMCU_APB1_GRP1_WWDG_STOP);
 #endif /* CONFIG_SOC_SERIES_STM32H7X */

